### ISSUE TITLE
Media (motion)

### ISSUE THREAD

**Issue by penpadstudio** [NHSLeadership/nightingale#217]
Create a media motion box without transcript card.

**Comment by cehwitham** [comment]
Can you provide a screenshot or some more details please @Pro-Paul ?

**Comment by penpadstudio** [comment]
This is what we have built:

![screen shot 2018-04-11 at 15 50 04](https://user-images.githubusercontent.com/23170050/38624465-0ab32eb0-3da0-11e8-8373-d30f76518881.png)

This is what I mean by no transcript box:

<img width="458" alt="screen shot 2018-04-11 at 15 51 02" src="https://user-images.githubusercontent.com/23170050/38624516-26695148-3da0-11e8-9bb5-8cce2124c684.png">

**Comment by cehwitham** [comment]
So is this achieved by just not including the transcript box markup or is the style of the video component is different when there's no transcript box? It looks from this screenshot as if there's a drop shadow on it?

**Comment by penpadstudio** [comment]
Yes, so basically all that is needed is to remove the transcript box from the markup for this component.
Ignore the drop shadow as this only shows because I screenshot from sketch design file.

**Comment by cehwitham** [comment]
So I think we can either close this issue as there's no new development to be done, or add an example to the demo page. What do you think @Pro-Paul ?

**Comment by penpadstudio** [comment]
@cehwitham @Android63 Add example to demo page please. Thanks
